go.mod

```diff
@@ -69,12 +69,192 @@ require (
 	oras.land/oras-go/v2 v2.0.0-rc.6
 )
 
+require (
```

Comment:
I'm wondering where all of these come from. What version of Go do you have installed? Could you please run `go mod tidy` and see if these entries change?